_posts/2024-03-20-strimzi-kraft-migration.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title:  "Migrate your Strimzi-operated cluster from ZooKeeper to KRaft"
+date: 2024-03-20
+author: paolo_patierno
+---
+
+In the previous [blog post](TBD) we explored how ZooKeeper is used to store Kafka clusters metadata and what are the current limitations.
+We introduced the new KRaft protocol which allows to overcome such limitations and store metadata in Kafka itself, taking into account that the ZooKeeper support will be removed soon.
+For this reason, there is the need to migrate existing clusters from using ZooKeeper to KRaft for storing metadata.
+We also covered the overall procedure to do that.
+Such procedure is not that simple because of the several configuration changes to be made on the cluster nodes, as well as the multiple times the nodes have to be rolled.
+But if your cluster is running on Kubernetes and it is Strimzi-operated, we got you covered!
+In this blog post, we are going to see how the Strimzi cluster operator provides a semi-automated process to run the migration easily, forgetting all the manual steps you should be doing to reach the same goal.
```

Comment:
```suggestion
The good news is that, if your cluster is powered by Strimzi, this process is much easier, and can be driven by simply annotating your Kafka resource.
You don't need to worry about updating configurations and restarting cluster nodes, as this is all automated.
```

Reply:
I used part of the suggestions mixing with my content.